### PR TITLE
openbmc rflash command support for -l --list option

### PR DESCRIPTION
**Pull request by gurevichmark**
Implements issue #3157 

List in a table format each uploaded update file.

Single endpoint:
```
[root@stratton01 xcat]# rflash p9euh02 -l
p9euh02: ID       Purpose State    Version
p9euh02: -------------------------------------------------------
p9euh02: deb46041 BMC     Active   v1.99.6-71-g0980d39
p9euh02: b7389829 Host    Ready    IBM-witherspoon-ibm-OP9_v1.16_1.24
p9euh02:
```

Two endpoints:
```
[root@stratton01 xcat]# rflash p9euh02,p9euh01 --list
p9euh01: ID       Purpose State    Version
p9euh01: -------------------------------------------------------
p9euh01: 6584ef1  BMC     Invalid  v1.99.6-19-gab8de5b
p9euh01: b7389829 Host    Ready    IBM-witherspoon-ibm-OP9_v1.16_1.24
p9euh01:
p9euh02: ID       Purpose State    Version
p9euh02: -------------------------------------------------------
p9euh02: deb46041 BMC     Active   v1.99.6-71-g0980d39
p9euh02: b7389829 Host    Ready    IBM-witherspoon-ibm-OP9_v1.16_1.24
p9euh02:
```

### CODE REVIEW

File: xCAT-server/lib/xcat/plugins/openbmc.pm [hunk 1]
```diff
@@ -483,7 +490,7 @@ sub parse_args {
             else {
                 # Filename was not passed, check flags allowed without file
                 if ($opt !~ /^-c$|^--check$|^-l$|^--list/) {
-                    return ([ 1, "Unsupported no file command command option: $opt" ]);
+                    return ([ 1, "Unsupported no file command option: $opt" ]);
```
Comment (whowutwut) [inline xCAT-server/lib/xcat/plugins/openbmc.pm]: This message is a little strange to understand... So this is for when we pass in a file to rflash.   `rflash <node> <file>` 

For the line above at 487, should we say:  `Invalid option specified when a file is provided: $opt` 
And for this line: ``Invalid option specified: $opt`

File: xCAT-server/lib/xcat/plugins/openbmc.pm [hunk 2]
```diff
@@ -204,6 +204,13 @@ my %status_info = (
     RVITALS_RESPONSE => {
         process        => \&rvitals_response,
     },
+    RFLASH_LIST_REQUEST  => {
```
Comment (whowutwut) [inline xCAT-server/lib/xcat/plugins/openbmc.pm]: I think here we decided to put the commands in alphabetical order...  so should go up in the list.

File: xCAT-server/lib/xcat/plugins/openbmc.pm [hunk 3]
```diff
@@ -693,8 +700,9 @@ sub parse_command_status {
             $next_status{RINV_FIRM_REQUEST} = "RINV_FIRM_RESPONSE";
         }
         if ($list) {
-            xCAT::SvrUtils::sendmsg("List option is not yet supported.", $callback);
-            return 1;
+            #Display firmware update files uploaded to BMC
```
Comment (whowutwut) [inline xCAT-server/lib/xcat/plugins/openbmc.pm]: Add a space after `#` for consistency.

**Comment by whowutwut** [comment]
:+1:  I like the formatting of the output, looks nice.

**Comment by gurevichmark** [comment]
Suggested review changes implemented.

**Comment by cxhong** [comment]
Victor is on vacation next couple days.   Mark made changes as from Victor's requested.  I will merge it.

**Pull request merged**